src/DeviceInput/Implementations/webDeviceInputSystem.ts

```diff
@@ -521,10 +521,10 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
                 deviceEvent.previousState = previousButton;
                 deviceEvent.currentState = pointer[evt.button + 2];
 
-                if (deviceType === DeviceType.Mouse && this._mouseId >= 0 && this._elementToAttachTo.hasPointerCapture(this._mouseId)) {
+                if (deviceType === DeviceType.Mouse && this._mouseId >= 0 && this._elementToAttachTo.hasPointerCapture?.(this._mouseId)) {

```

Comment:
Is it okay to not call releasePointerCapture when setPointerCapture is called?

Reply:
It should automatically be released on a `pointerout` event but just in case there are issues with calling `setPointerCapture`, I've added a check to make sure that we only set it if `hadPointerCapture` is present.

Reply:
I'm not sure I understand. If it will autorelease on `pointerout`, why do we need to call this at all here?